misc/Makefile

```diff
@@ -1,12 +1,34 @@
 include ../Make.defs
 
-all clean:
+VERSION != grep 'Version:' ../tcb.spec | sed -e 's!Version:[ \t]*!!g'
```

Comment:
There is a common pattern for such things:
```
VERSION = $(shell sed '/^Version: */!d;s///;q' ../tcb.spec)
```

Reply:
Fixed in rebased commit.

Reply:
FYI, I've used `!=` for shell invokation intentionally, as `$(shell cmd args…)` is a GNU-specific extension, and is AFAIK less portable.